pycytominer/cyto_utils/cells.py

```diff
@@ -513,6 +520,7 @@ def aggregate_compartment(
         object_dfs = []
         for compartment_df in self._compartment_df_generator(
             compartment=compartment,
+            features=features,
```

Comment:
I'm not sure where this is used?

Reply:
Should be clear now after the fix, lmk if not

Reply:
I think it was clear even before the fix - sorry for overlooking!